package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aserto/aserto-react-components",
-  "version": "7.2.2",
+  "version": "7.2.3",
```

Comment:
this should be a minor version bump as it introduces new functionality library consumers